sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -8,6 +8,23 @@ toc::[]
 
 You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
 
+[IMPORTANT]
+====
+When upgrading OpenShift Container Platform 4.10 with OpenShift sandboxed containers 1.2 to OpenShift Container Platform 4.11 with OpenShift sandboxed containers 1.3, the recommended path is first to upgrade OpenShift sandboxed containers to 1.3, and then to upgrade OpenShift Container Platform to to 4.11.
+
+After the upgrade of OpenShift sandboxed containers, new features and bug fixes in the Operator code are available immediately. However, additional steps are required to upgrade certain resources.
+
+* To get the new features and bug fixes for the Kata runtime, delete and re-create the `KataConfig` resource. This must be performed after the cluster upgrade, to deploy the new RHCOS extension (otherwise the old Kata RPMs will be used), and prior to the OpenShift Container Platform upgrade, because the Kata runtime is shipped as an RHCOS extension.
```

Comment:
I was wrong... new features of the Kata runtime automatically come along with the OCP upgrade since these are RHCOS extensions ! No need to delete/re-create the KataConfig... sorry for the confusion (too many moving pieces).